modules/azure/src/main/java/org/testcontainers/containers/CosmosDBEmulatorContainer.java

```diff
@@ -0,0 +1,99 @@
+package org.testcontainers.containers;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+import static org.testcontainers.containers.Constants.EMULATOR_CERTIFICATE_FILE_NAME;
+import static org.testcontainers.containers.Constants.EMULATOR_KEY;
+import static org.testcontainers.containers.Constants.KEYSTORE_FILE_NAME;
+import static org.testcontainers.containers.Constants.STORE_PASSWORD;
+import static org.testcontainers.containers.Constants.STORE_TYPE;
+import static org.testcontainers.containers.Constants.TEMP_DIRECTORY_NAME;
+import static org.testcontainers.utility.DockerImageName.parse;
+
+/**
+ * An Azure CosmosDB container
+ *
+ * Default port is 8081.
```

Comment:
```suggestion
```

Better to omit this specific detail here, especially since a user will obtain the mapped port programatically.

Reply:
👍 Added to my patch list.